docs/architecture/seo.md

```diff
@@ -49,50 +50,59 @@ export interface PageMeta {
 }
 ```
 
-HTML5 supports a number of specific metatag properties, such as `description`, `keyworks`, etc. These metatags are used by search engines, social platforms and bots. In order to be more specific on one or the other platform, some social platform introduce a specific set of properties. Facebook for example uses an Open Graph protocol that enables any web page to become a rich object in a social graph. Specific metatags can be used to articulate the experience on the social platform. The code snippet below shows a custom page description for facebook.
+HTML5 supports a variety of meta tag properties, such as `description`, `keywords`, and so on. These meta tags are used by search engines, social platforms and bots. Some social platforms have introduced their own sets of properties that are specific to their platforms. For example, Facebook uses an Open Graph protocol that enables any web page to become a rich object in a social graph. Specific meta tags can be used to articulate the experience on the social platform. The code snippet below shows a custom page description for Facebook:
 
 ```html
-<meta property="og:title" content="Custom title for facebook" />
-<meta property="og:description" content="Custom description for facebook" />
+<meta property="og:title" content="Custom title for Facebook" />
+<meta property="og:description" content="Custom description for Facebook" />
 ```
 
 ### Page Meta Resolvers
-In order to support the potential multitude of meta tags, Spartacus uses a small framework to customize and extend the meta tags per page. 
 
-Spartacus is shipped with `pageMetaResolvers` which resolves the page meta data for a specific page. Some of the meta data might be driven by CMS data, but most meta tags are computed based on product content, category content, search results,etc.
+To support the potential multitude of meta tags, Spartacus uses a small framework to customize and extend the meta tags per page.
+
+Spartacus ships with `pageMetaResolvers` that resolve the page meta data for a specific page. Some of the meta data might be driven by CMS data, but most meta tags are computed based on product content, category content, search results, and so on.
+
+The page meta data is updated dynamically during navigation, but can be delivered statically using SSR.
+
+### Title Resolver
+
+Adding an HTML `title` tag to your page has the following advantages:
 
-The page meta will be updated dynamically during navigation, but can be delivered statically using SSR.
+- the page can be uniquely addressed in the browser (that is, through the browser history, bookmarks, tabs, and so on)
```

Comment:
we have too much space between the bullets I believe

Reply:
I tend to find the extra space makes things easier to read, but I see that it could also look like the points don't have enough "connection" to each other. I removed the spacing, at your request 🙂